examples/sample-allOption.yml

```diff
@@ -0,0 +1,72 @@
+id: myapp
+name: MyApp Service (powered by WinSW)
+description: This service is a service created from a sample configuration
+executable: java
+caption: WinSW
+#serviceaccount:
+#    domain: YOURDOMAIN
+#    user: useraccount
+#    password: Pa55w0rd
+#    allowservicelogon: yes
+#failureActions:
```

Comment:
I don't think `failureActions` is better than `onFailure`.

Reply:
will change into `onFailure`